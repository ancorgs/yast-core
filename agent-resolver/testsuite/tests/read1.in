# /etc/resolv.conf
#
# Automatically generated by SuSEconfig on Mon Aug 21 13:10:36 CEST 2000.
#
# PLEASE DO NOT EDIT THIS FILE!
#
# Change variables (NAMESERVER + SEARCHLIST) in /etc/rc.config instead.
#
#
search suse.de suse.de
nameserver 10.10.0.1
nameserver 10.10.128.1
domain  suse.de
sortlist        130.155.160.0/255.255.240.0 130.155.0.0
options ndots:3 rotate no-check-names
