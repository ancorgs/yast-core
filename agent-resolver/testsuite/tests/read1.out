($["domain":"suse.de", "nameserver":["10.10.0.1", "10.10.128.1"], "options":["ndots:3", "rotate", "no-check-names"], "search":["suse.de", "suse.de"], "sortlist":["130.155.160.0/255.255.240.0", "130.155.0.0"]])
